setup.py

```diff
@@ -35,7 +35,7 @@ def find_version(fname):
 history = open('HISTORY.rst').read().replace('.. :changelog:', '')
 requirements = [
     str(requirement.req)
-    for requirement in parse_requirements('requirements.txt')
+    for requirement in parse_requirements('requirements.txt', session=uuid.uuid1())
```

Comment:
uuid.uuid1() is a bad precedent to set as it will fail if, for any reason, pip tries to actually use it.